src/lib/y2network/interface.rb

```diff
@@ -21,6 +21,8 @@ module Y2Network
   class Interface
     # @return [String] Device name (eth0, wlan0, etc.)
     attr_accessor :name
+    # @return [String]
+    attr_accessor :ethtool_options
```

Comment:
really all interfaces have to have this options?

Reply:
No, they haven't. We have to look a better place to add it, but so far we only have specialized classes for connection apart of the virtual and physical Interface subclasses. 

I will remove this option by now or will just update the comments, first adding a FIXME or TODO label, and permitting nil in the ones that are not generic / mandatory.